stellar-captive-core/README.md

```diff
@@ -0,0 +1,14 @@
+# Stellar Captive Core
+Captive Core is a deployment that uses a stellar core node to aid horizon ingest.  If you run Horizon with Captive Stellar-Core ingestion enabled Horizon will spawn a Stellar-Core subprocess. Horizon's ingestion system will then stream ledgers from the subprocess via a filesystem pipe. The disadvantage of running both Horizon and the Stellar-Core subprocess on the same machine is it requires detailed per-process monitoring to be able to attribute potential issues (like memory leaks) to a specific service.
```

Comment:
> The disadvantage of running both Horizon and the Stellar-Core subprocess on the same machine is it requires detailed per-process monitoring to be able to attribute potential issues (like memory leaks) to a specific service.

I don't think this belongs in the docs. But also if things go wrong standard linux tooling, like top, should be enough in establishing which processes takes resources.

Reply:
I took this from the docs in the go monorepo because I didn't know what else to put here.  That blurb is written by the developers as explanation as to why they created the HTTP API.

Reply:
Maybe it's worth linking to the docs then to avoid duplication?